Makefile

```diff
@@ -0,0 +1,41 @@
+all: coverageclean test coverage quality
+
+coverageclean:
+	rm -fr .coverage
+
+clean: coverageclean typecoverageclean
+	FILES=$$(find . -name \*.pyc); for f in $${FILES}; do rm $$f; done
+
+test:
+	ENV=test nosetests --exclude='tests/integration' --cover-package=sqla_vertica_python --with-coverage --with-xunit --cover-html --cover-xml --nocapture --cover-inclusive
+
+citest:
+	ENV=test nosetests --exclude='tests/integration' --cover-package=sqla_vertica_python --with-coverage --with-xunit --cover-html --cover-xml --nocapture --cover-inclusive --xunit-file=test-reports/junit.xml
+
+coverage:
+	python setup.py coverage_ratchet
+
+cicoverage: coverage
+	@echo "Looking for un-checked-in unit test coverage metrics..."
+	@git status --porcelain metrics/coverage_high_water_mark
+	@test -z "$$(git status --porcelain metrics/coverage_high_water_mark)"
+
+flake8:
+	flake8 bluelabs_joblib tests
```

Comment:
Package name needs updating? (Can’t propose a change on the mobile github ☹️)

Reply:
Fixed - thanks!